### ISSUE TITLE
license mismatch

### ISSUE THREAD

**Issue by 5cat** [qwertyquerty/pypresence#185]
Hey,

The license in the root of the repo is BSD-3 while the one in the setup.py is MIT.
I wanted to create a PR but not sure which one is the correct one, will leave that up to you.

**Comment by farchord** [comment]
Yeah I'm having the same issue. Trying to package this module for Fedora, and the different licenses are making it difficult.

the setup.py license is saying that the module is using the MIT license, but the LICENSE file says BSD-3-Clause (https://github.com/qwertyquerty/pypresence/blob/master/LICENSE)

Now, MIT license is also used for the jquery and underscore javascript libraries but the docs folder can simply be omitted when we package the library.

**Comment by qwertyquerty** [comment]
MIT
